### ISSUE TITLE
Automatically generate RST documentation for wsdl files

### ISSUE THREAD

**Issue by mvantellingen** [mvantellingen/python-zeep#110]
Zeep should be able to generate API documentation for a WSDL with examples etc

**Comment by devdavidkarlsson** [comment]
Will this be able to generate xml request templates? We need the templates for our soap client, where we populate templates with variables and call the soap service operations using these filled out xml templates

**Comment by VMahipalReddy** [comment]
+1

**Comment by EmpireofKings** [comment]
bump

This would be a great feature addition, IMHO it should take priority over all other feature requests.

**Comment by mvantellingen** [comment]
Yeah would be pretty great. Can you submit a PR for this? I'll happily check it :-)
